### PR TITLE
Migration for region normalization was added

### PR DESCRIPTION
**Pull request by DaniyarAskarov**
Migration normalize region relations with countries and agencies for markups usage

### CODE REVIEW

**Comment by DaniyarAskarov** [comment]
Issue: https://github.com/happy-travel/agent-app-project/issues/1245

File: HappyTravel.Edo.Data/Migrations/20220321164123_RegionNormalization.cs [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+                {
+                    "DZ", "AO", "BJ", "BW", "BF", "BI", "CM", "CV", "CF",
+                    "TD", "KM", "CG", "CD", "CI", "DJ", "GQ", "ER", "ET",
+                    "GA", "GM", "GH", "GN", "GW", "KE", "LS", "LR", "LY",
+                    "MG", "MW", "ML", "MR", "MU", "YT", "MA", "MZ", "NA",
+                    "NE", "NG", "RE", "RW", "SH", "ST", "SN", "SC", "SL",
+                    "SO", "ZA", "SS", "SD", "SZ", "TZ", "TG", "TN", "TN",
+                    "EH", "ZM", "ZW"
+                }),
+            (new InternalRegion("Far east"), new []
```
Comment (AlexeyShok) [inline HappyTravel.Edo.Data/Migrations/20220321164123_RegionNormalization.cs:93]: East with a capital letter

File: HappyTravel.Edo.Data/Migrations/20220321164123_RegionNormalization.cs [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+                {
+                    "IO", "BN", "KH", "CN", "TF", "HK", "ID", "JP", "LA",
+                    "MO", "MY", "MN", "MM", "KR", "PH", "PW", "SG", "KP",
+                    "TW", "TH", "TL", "VN"
+                }),
+            (new InternalRegion("Gcc"), new []
```
Comment (AlexeyShok) [inline HappyTravel.Edo.Data/Migrations/20220321164123_RegionNormalization.cs:99]: GCC

File: HappyTravel.Edo.Data/Migrations/20220321164123_RegionNormalization.cs [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+                    "AM", "AZ", "BY", "EE", "GE", "KZ", "KG", "LV", "LT",
+                    "MD", "RU", "TJ", "TM", "UA", "UZ"
+                }),
+            (new InternalRegion("Oceania"), new []
+                {
+                    "AS", "AU", "CX", "CC", "CK", "FJ", "PF", "GU", "HM",
+                    "KI", "MH", "FM", "NR", "NC", "NZ", "NU", "NF", "MP",
+                    "PG", "PN", "WS", "SB", "TK", "TO", "TV", "VU", "WF"
+                }),
+            (new InternalRegion("North america"), new []
```
Comment (AlexeyShok) [inline HappyTravel.Edo.Data/Migrations/20220321164123_RegionNormalization.cs:131]: America with a capital letter

File: HappyTravel.Edo.Data/Migrations/20220321164123_RegionNormalization.cs [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+                {
+                    "AQ", "BM", "CA", "GL", "PM", "US", "UM"
+                }),
+            (new InternalRegion("Latin america"), new []
```
Comment (AlexeyShok) [inline HappyTravel.Edo.Data/Migrations/20220321164123_RegionNormalization.cs:135]: America with a capital letter

**Review by kirillta — APPROVED** [review]
LGTM, except @AlexeyShok comments: English names and titles write with starting caps for every word

**Review by kirillta — APPROVED** [review]

**Review by kirillta — APPROVED** [review]

**Comment by DaniyarAskarov** [comment]
Already in master

**Pull request closed without merging**